components/treeview/stories/treeview.stories.js

```diff
@@ -0,0 +1,241 @@
+Default.args = {
+  items: [
+    {
+      id: "label1",
+      label: "Label 1",
+      link: "#",
+      isSelected: true,
+    },
+    {
+      id: "group1",
+      label: "Group 1",
+      link: "#",
+      isOpen: true,
+      items: [
+        {
+          id: "label2",
+          label: "Label 2",
+          link: "#",
+          isDisabled: true
+        },
+        {
+          id: "label3",
+          label: "Label 3",
+          link: "#"
+        },
+      ]
+    },
+    {
+      id: "group2",
+      label: "Group 2",
+      link: "#",
+      items: [
+        {
+          id: "label3",
+          label: "Label 3",
+          link: "#"
+        },
+        {
+          id: "group3",
+          label: "Group 3",
+          link: "#",
+          items: [
+            {
+              id: "label4",
+              label: "Label 4",
+              link: "#"
+            },
+          ]
+        },
+      ]
+    }
+  ]
+};
+
+export const FoldersAndFiles = Template.bind({});
+FoldersAndFiles.args = {
+  items: [
+    {
+      id: "label1",
+      label: "Label 1",
+      link: "#",
+      icon: "Document",
+    },
+    {
+      id: "group1",
+      label: "Group 1",
+      link: "#",
+      isOpen: true,
+      isSelected: true,
+      icon: "Folder",
+      items: [
+        {
+          id: "label2",
+          label: "Label 2",
+          link: "#",
+          icon: "Document",
+        },
+        {
+          id: "label3",
+          label: "Label 3",
+          link: "#",
+          icon: "Document",
+        },
+      ]
+    },
+    {
+      id: "group2",
+      label: "Group 2",
+      link: "#",
+      icon: "Folder",
+      items: [
+        {
+          id: "label3",
+          label: "Label 3",
+          link: "#",
+          icon: "Document",
+        },
+        {
+          id: "group3",
+          label: "Group 3",
+          link: "#",
+          icon: "Folder",
+          items: [
+            {
+              id: "label4",
+              label: "Label 4",
+              link: "#",
+              icon: "Document",
+            },
+          ]
+        },
+      ]
+    }
+  ]
+};
+
+export const Thumbnails = Template.bind({});
+Thumbnails.args = {
+  items: [
+    {
+      id: "group1",
+      label: "Group 1",
+      link: "#",
+      isOpen: true,
+      thumbnail: {
+        imageURL: "img/thumbnail.png",
+        altText: "Woman crouching"
+      },
+      items: [
+        {
+          id: "label2",
+          label: "Label 2",
+          link: "#",
+          thumbnail: {
+            imageURL: "img/thumbnail.png",
+            altText: "Woman crouching"
+          }
+        },
+        {
+          id: "label3",
+          label: "Label 3",
+          link: "#",
+          thumbnail: {
+            imageURL: "img/flowers.png",
+            altText: "Flowers"
+          }
+        },
+      ]
+    },
+  ]
+};
+
+export const WithSections = Template.bind({});
+WithSections.args = {
+  items: [
```

Comment:
Maybe add a second section of data here like on the docs site, so we can test that they're spaced the same when within the same parent.